### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by praveentiru** [lucamug/elm-starter#13]
Hi,

I was trying this out on Windows 10 in Git Shell. I have run into the following issues:
1. Starter tries to create a symlink for certain assets - Unless windows shell has admin privileges, script cannot create symlink
2. spawn to create child processes fails - Adding `shell: true` to `startCommand` in elm-starter fixes this issue
3. serverDev and watchStartElm are not recognized as commands - I get the following error after fixing the issues listed above:
         `~elm\material-front-end/src-elm-starter/elm-starter' is not recognized as an internal or external command`

I believe for some reason the command getting executed is elm-starter serverDev rather than node elm-starter serverDev etc. I am unable to figure out why this is happening in Windows. I can work on a patch for the same with little guidance.

**Comment by lucamug** [comment]
Yes, I never tested in Windows. If anypody want to submit a PR I will look into it
